### PR TITLE
first commit, created new repository

### PR DESCRIPTION
**Pull request by Staroz**

### CODE REVIEW

File: cypress/e2e/testTrello.cy.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        cy.visit('https://trello.com')
+        cy.get('.Buttonsstyles__ButtonGroup-sc-1jwidxo-3 > [href="/login"]').click()
+        cy.get('#user').type('marcourquidi17@outlook.com')
+        cy.get('#login').click()
+        cy.origin('https://id.atlassian.com',()=>{
+            cy.get('#password').type('Hijodecain17')
+            cy.get('#login-submit').click()
+            })
+        });
+})
```
Comment (sphinix27) [inline cypress/e2e/testTrello.cy.js:24]: This file is not required, it can be removed.

Comment (sphinix27) [reply cypress/e2e/testTrello.cy.js:24]: After removing this file, this PR can be approved.

**Review by sphinix27 — CHANGES_REQUESTED** [review]

File: cypress/e2e/testTrello.cy.js (hunk 1 above)
Comment (Staroz) [reply cypress/e2e/testTrello.cy.js:24]: ok, this file was erased. please review it again please.

**Comment by Staroz** [comment]
this files was erased.

**Review by sphinix27 — APPROVED** [review]

**Pull request merged**